man/C_run_testthat_tests.Rd

```diff
@@ -0,0 +1,16 @@
+% Generated by roxygen2: do not edit by hand
```

Comment:
Should these `.RD` files be checked in the Github?

Reply:
devtools::build() or devtools::install() does not invoke devtools::document(). (Invoking devtools::document() creates the .Rd files from the Roxygen2 documentation in the .R files).
But devtools::install uses these .RD files for installing the documentation of the package. As far as I understood, it is/was also practice to write those .Rd files by hand.

But maybe it would be good to create a Github actions which calls devtools::document()  and verifies that nothing has changed! => If the developer forgot to call devtools::document() this GH actions fails! So everybody is forced to keep the .Rd files up-to-date, what do you think?

Reply:
Thanks for explanation @tomuben! 
That is fine from my side. I was not sure if they really need to be checked in for CRAN submission or can also be omitted.

Reply:
I created https://github.com/exasol/r-exasol/issues/84 to fix this in the future!